server/api/hook.go

```diff
@@ -191,7 +191,7 @@ func PostHook(c *gin.Context) {
 		msg := "failure to parse yaml from hook"
 		log.Debug().Err(err).Str("repo", repo.FullName).Msg(msg)
 		c.String(http.StatusBadRequest, msg)
-		return
+		// return
```

Comment:
Is this change related to #811?

Reply:
Huch, did not mean to commit that here, but yes this would be my proposed solution to #811, that should be all it takes

Reply:
Shall I revert this ?

Reply:
Yeah, lets move it into a separate PR.

Reply:
✅